openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/TableSummary/TableSummary.component.tsx

```diff
@@ -40,21 +36,25 @@ import { TableProfilerTab } from '../../../Database/Profiler/ProfilerDashboard/p
 import './table-summary.less';
 import { TableSummaryProps } from './TableSummary.interface';
 
-function TableSummary({ entityDetails: tableDetails }: TableSummaryProps) {
+function TableSummary({
+  entityDetails: tableDetails,
+  permissions,
+}: Readonly<TableSummaryProps>) {
   const { t } = useTranslation();
   const location = useCustomLocation();
   const history = useHistory();
   const isTourPage = location.pathname.includes(ROUTES.TOUR);
-  const { getEntityPermission } = usePermissionProvider();
 
   const [testSuiteSummary, setTestSuiteSummary] = useState<TestSummary>();
   const [tablePermissions, setTablePermissions] = useState<OperationPermission>(
     DEFAULT_ENTITY_PERMISSION
   );
-
+  useEffect(() => {
+    setTablePermissions(permissions as OperationPermission);
+  }, [permissions]);
```

Comment:
If we have permission we can directly use it instead setting it in state on a useEffect